camp/ocampbell/xBoxTriggerProject/src/main/java/frc/robot/RobotContainer.java

```diff
@@ -0,0 +1,31 @@
+// ************************************************************
+// Bishop Blanchet Robotics
+// Home of the Cybears
+// File: RobotContainer.java
+// ************************************************************
+
+// ʕ •ᴥ•ʔ ʕ•ᴥ•  ʔ ʕ  •ᴥ•ʔ ʕ •`ᴥ´•ʔ ʕ° •° ʔ ʕ •ᴥ•ʔ ʕ•ᴥ•  ʔ ʕ  •ᴥ•ʔ ʕ •`ᴥ´•ʔ ʕ° •° ʔ 
+
+package frc.robot;
+
+import frc.robot.control.SubsystemCollection;
+import frc.robot.subsystems.ControlledFalcon;
+
+/**
+ * This class is where the bulk of the robot should be declared. Since Command-based is a
+ * "declarative" paradigm, very little robot logic should actually be handled in the {@link Robot}
+ * periodic methods (other than the scheduler calls). Instead, the structure of the robot (including
+ * subsystems, commands, and trigger mappings) should be declared here.
+ */
+public class RobotContainer {
+
+  private SubsystemCollection subsystems;
+
+  public RobotContainer() {}
+
+
+  public void teleopPeriodic() {
+    subsystems.setFalconSubsystem(new ControlledFalcon());
```

Comment:
Subsystem initialization usually is done in the constructor of RobotContainer.  Subsystems ideally should not be re-initialized.

Adding new objects in a periodic will tend keep the processor and memory thrashing an equivalent object in and out of scope.  Also will be a drag on garbage collector.